ISSUE_TEMPLATE/new_meetup.md

```diff
@@ -0,0 +1,26 @@
+***Please help us by filling out the text below. All answers are optional***
+
+# New Meetup
+
+## Where
+
+*A short description of the event. Please also add any links, e.g. to the meetup page.*
+
+## When
+
+*the date. please name the timezone*
+
+[ ] I'd like the event to be public calendar
+[ ] It's a repeat event, I would like to have access to the calender to do things on my own
+
+## Support
+
+[ ] I need coaching
+[ ] I need contacts in my area
+[ ] I need help promoting
```

Comment:
I'm a bit unclear on what this entails. Are we announcing the event for the organizers on different channels? That actually sounds like a nice thing. We should have a list of things we can offer up-front, to make things clear for the organizer and for us ("help promomoting" is a bit fuzzy). E.g.

```
You can promote your event by advertising it on Twitter, Reddit and other channels. We understand that the breadth of this may be overwhelming for organizers and would offer to advertise the event for you. If you already have an active Twitter account and/or regularly announce your meetup through other channels you might not need this option.

[ ] I need additional help promoting my event
```